src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -328,7 +351,8 @@ blockToMarkdown' opts (Div attrs ils) = do
   variant <- asks envVariant
   return $
     case () of
-         _ | isEnabled Ext_fenced_divs opts &&
+         _ | variant == Markua -> contents <> blankline
```

Comment:
It would be possible to use Divs with special classes to produce Markua blurbs and asides.  Would that make sense?

Reply:
We looked into the idea and it's now possible to create blurbs and asides by providing a div with class "blurb" or "aside".

Reply:
Have tests for this feature been added?

Reply:
we now added testcases